protocols/v2/noise-sv2/src/signature_message.rs

```diff
@@ -1,14 +1,39 @@
+// # Signature-Based Messages Handling
+//
+// Defines the [`SignatureNoiseMessage`] struct, which represents a signed message used in the
+// Noise protocol to authenticate and verify the identity of a party during the handshake. It
+// includes functionality for signing and verifying messages using Schnorr signatures with the
+// [`secp256k1`] elliptic curve.
```

Comment:
We should not reference modules outside of our crates, this should be Secp256k1

Reply:
cc: @rrybarczyk